README.md

```diff
@@ -1,70 +1,98 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Hotel Booking app Front-end
 
-## Available Scripts
+### Back-end API access on:
+    https://github.com/RokoVarano/hotel_services_backend
 
-In the project directory, you can run:
+> React Frontend of the project
 
-### `yarn start`
+![Login](./src/components/assets/login.png)
+![Services](./src/components/assets/services.png)
+![Service](./src/components/assets/service.png)
+![Reserve](./src/components/assets/reserve.png)
+![Delete](./src/components/assets/delete.png)
```

Comment:
don't forget that the application needs to look exactly like design with exception of photos, titles and descriptions